dronekit/__init__.py

```diff
@@ -889,6 +954,14 @@ def listener(self, name, m):
             self._mount_yaw = m.pointing_c / 100
             self.notify_attribute_listeners('mount', self.mount_status)
 
+        self._capabilities = 0
+        self._raw_version = 0
+
+        @self.on_message('AUTOPILOT_VERSION')
```

Comment:
Maybe I'm understanding wrong, but if the `on_message` is in the `Capabilities` object, then the listener won't start running until after the object is instantiated (i.e. when somebody tries check a capability).  Since the autopilot will only send one `AUTOPILOT_VERSION` (rather than sending it multiple times per second), we need that listener to be running from the start to catch it, not after the user requests the data.